util/Migrator/DbScripts/2023-11-15_00_TurnOffSecretsManagerBeta.sql

```diff
@@ -0,0 +1,16 @@
+IF COL_LENGTH('[dbo].[Organization]', 'SecretsManagerBeta') IS NOT NULL
+BEGIN
+    -- Set AccessSecretsManager to 0 for Organization Users in SM Beta orgs
+    UPDATE [dbo].[OrganizationUser]
+    SET AccessSecretsManager = 0
+    FROM [dbo].[OrganizationUser] AS ou
+    INNER JOIN [dbo].[Organization] AS o ON ou.OrganizationId = o.Id
```

Comment:
⛏️ Strange syntax to me with `AS` and I don't think it's needed.

Reply:
The T-SQL expression IF COL_LENGTH('[dbo].[Organization]', 'SecretsManagerBeta') IS NOT NULL is used to check if a column named SecretsManagerBeta exists in the Organization table within the dbo schema.

if not, it will not run

Reply:
AS is extraneous, it is not needed does no harm besides not possibly meeting a coding standard